lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -400,26 +400,60 @@ export class BrowserCacheManager extends CacheManager {
      * Gets the active account
      */
     getActiveAccount(): AccountInfo | null {
-        const activeAccountIdKey = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
-        const activeAccountId = this.browserStorage.getItem(activeAccountIdKey);
-        if (!activeAccountId) {
+        const activeAccountKeyNew = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT_NEW);
+        const activeAccountValueNew = this.getItem(activeAccountKeyNew);
+        if (!activeAccountValueNew) {
+            const activeAccountKeyOld = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
+            const activeAccountValueOld = this.getItem(activeAccountKeyOld);
+            if(!activeAccountValueOld)
+            {
+                return null;
+            }
+            const activeAccount = this.getAccountInfoByFilter({localAccountId: activeAccountValueOld})[0] || null;
+            if(activeAccount)
+            {
+                this.setActiveAccount(activeAccount);
+                this.browserStorage.removeItem(activeAccountKeyOld);
+                return activeAccount;
+            }
+            return null;
+        }
+        else
+        {
+            const activeAccountValueObj = this.validateAndParseJson(activeAccountValueNew);
```

Comment:
```suggestion
            const activeAccountValueObj = this.validateAndParseJson(activeAccountValueNew) as AccountInfo;
```

Then you shouldn't need `hasOwnProperty`, square bracket notation, below.